### PR TITLE
feat(configuration): added default configuration value for category b…

### PR DESCRIPTION
**Pull request by Dreimus**
If the product is accessed from another category than product default category, it will generate the breadcrumb according to current category.

| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | By default, product breadcrumb is generated by using product default category. With this feature, if a category is stored on the product controller and assigned to the viewed product, the breadcrumb will be generated accordingly. 
| Type?             | improvement 
| Category?         | FO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket? | 
| Related PRs       | https://github.com/PrestaShop/PrestaShop/pull/30546
| How to test?      | Upgrade to 9.0.0 branch or run the query manually.

### CODE REVIEW

**Comment by Dreimus** [comment]
Ping @kpodemski

**Review by Hlavtox — APPROVED** [review]

**Comment by Hlavtox** [comment]
We can merge after https://github.com/PrestaShop/PrestaShop/pull/30546 is merged. 👍

**Comment by ga-devfront** [comment]
PR https://github.com/PrestaShop/PrestaShop/pull/30546 is merged. Need to be tested by QA before merging it.

**Review by ga-devfront — APPROVED** [review]

**Comment by kpodemski** [comment]
@PrestaShop/qa-functional considering it only sets the default value in the database, I don't think this PR requires any manual tests

**Comment by Robin-Fischer-PS** [comment]
Hi @kpodemski ,

Indeed it does not require any manual tests, I let you or any other maintainer check it and merge it :)

And thanks for the PR @Dreimus !

**Pull request merged**
